extensions/simpread/package.json

```
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "simpread",
+  "title": "SimpRead",
+  "description": "Search and Open SimpRead Unreader",
+  "icon": "logo@512.png",
+  "author": "kenshin",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "src",
+      "title": "SimpRead Config Setting",
+      "description": "Set local simpread_config.json path",
+      "mode": "view"
+    },
+    {
+      "name": "sr",
+      "title": "SimpRead Unreader",
+      "description": "List of SimpRead Unreader",
```

```suggestion
      "title": "Browse Posts",
      "subtitle": "SimpRead",
      "description": "Browse SimpRead Posts",
```
Something like above